modules/storage-persistent-storage-selinuxChangePolicy-namespace.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * storage/understanding-persistent-storage.adoc
+//* microshift_storage/understanding-persistent-storage-microshift.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="using_selinuxChangePolicy_namespace_{context}"]
+= Changing seLinuxChangePolicy at the namespace level
+
+After applying the desired setting for `seLinuxChangePolicy` at the namespace level, all subsequently created pods, and restarted pods, in that namespace inherit the setting. However, if desired, you can override the inherited `SELinuxChangePolicy` setting for individual pods. Setting `seLinuxChangePolicy` at the pod level overrides inheritance from the namespace level setting for that pod.
+
+.Prerequisites
+
+* Logged in to a running {product-title} cluster with administrator privileges.
+
+* Access to the {product-title} console.
+
+.Procedure
+
+To set `SELinuxChangePolicy` per namespace:
+
+. Select the desired namespace:
+
+.. Click *Administration* > *Namespaces*. 
+
+.. On the *Namespaces* page, click the desired namespace. The *Namespace details* page appears.
+
+. Add the `seLinuxChangePolicy` label to the namespace:
+
+.. On the *Namespace details* page, next to *Labels*, click *Edit*.
+
+.. In the *Edit labels* dialog, add the label `storage.openshift.io/selinux-change-policy` and set it equal to either:
++
+* `MountOption`: Specifies avoiding recursive relabeling of all files by attempting to mount the volume with the correct SELinux label directly using the -o context mount option, thus helping to avoid pod timeout problems.
```

Comment:
We need to mention this is supported only for RWOP volumes, RWO/RWX volume is TP.

Reply:
Added this to the general SELinux topic.